packages/core/nuxt-theme-module/theme/components/LoginModal.vue

```diff
@@ -15,6 +15,10 @@
     <transition name="sf-fade" mode="out-in">
```

Comment:
It's not related to this particular issue but if this modal is for both login and registration perhaps it should be just `AuthModal`